core/models.py

```diff
@@ -268,6 +279,57 @@ def save(self: "PostVote", *args: int, **kwargs: int) -> None:
         Post.objects.filter(pk=self.post.pk).update(up_votes=up_votes, down_votes=down_votes)
 
 
+class PostAward(models.Model):
+    REWARD_POINTS: ClassVar[dict[int, int]] = {
+        1: 15,  # Level 1 gives 15 points
+        2: 25,  # Level 2 gives 25 points
+        3: 50,  # Level 3 gives 50 points
+    }
+
+    REWARD_CHOICES: ClassVar[list[tuple[str, str]]] = []
+
+    for level, points in REWARD_POINTS.items():

```

Comment:
done, but please check if it is correct.
About:
`self.fields["choice"].choices = [choice for choice in self.fields["choice"].choices if choice[0] != ""]`
in forms.py
This code delete first option of IconRadioSelect. I dont know why Django generate one empty option before all awards choices. I checked- there is no empty choice generated. How to prevent django from generating "check nothing" radio select ?

Reply:
Done